include/zephyr/drivers/video-controls.h

```diff
@@ -107,18 +227,307 @@ enum video_power_line_frequency {
  */
 #define VIDEO_CID_CAMERA_CLASS_BASE 0x009a0900
 
-/** Adjustments of exposure time and/or iris aperture. */
+/** Enables automatic adjustments of the exposure time and/or iris aperture.
+ * Manual exposure or iris changes when it is not @ref VIDEO_EXPOSURE_MANUAL is undefined.
+ * Drivers should ignore such requests.
+ */
 #define VIDEO_CID_EXPOSURE_AUTO (VIDEO_CID_CAMERA_CLASS_BASE + 1)
-enum video_exposure_auto_type {
+enum video_exposure_auto {
```

Comment:
Maybe I should do it the other way around: keep the `_type` suffix every time...